src/chess/uciloop.cc

```diff
@@ -139,6 +139,9 @@ void UciLoop::RunLoop() {
       // Ignore empty line.
       if (command.first.empty()) continue;
       if (!DispatchCommand(command.first, command.second)) break;
+      else {
+        SendResponse(std::string("executed ") + command.first);
```

Comment:
It doesn't do what was required, what was needed is `info string` command.

Also the main thing to fix is error part, not successful execution.

For the success part, we (potentially) only need it for setting variables, not for all UCI commands.